src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp

```diff
@@ -150,7 +150,7 @@
             </div>
             <div class="col-md-7 col-md-offset-0">
                 <h5><i>Attendance Weights: </i></h5>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)